addons/godot-firebase/auth/auth.gd

```diff
@@ -162,7 +162,7 @@ func _set_config(config_json: Dictionary) -> void:
     _delete_account_request_url %= _config.apiKey
     _update_account_request_url %= _config.apiKey
 
-    connect("request_completed", self, "_on_FirebaseAuth_request_completed")
+    connect("request_completed", Callable(self, "_on_FirebaseAuth_request_completed"))
```

Comment:
I would rather see this as:

`request_completed.connect(_on_FirebaseAuth_request_completed)`